libs/input-mapping/src/File/Strategy/Local.php

```diff
@@ -12,21 +12,25 @@
 
 class Local extends AbstractStrategy implements StrategyInterface
 {
-    public function downloadFile(array $fileInfo, string $destinationPath, bool $overwrite, Client $client): void
-    {
+    public function downloadFile(
+        array $fileInfo,
+        string $sourceBranchId,
+        string $destinationPath,
+        bool $overwrite,
```

Comment:
Toto se částečně vrací k původní verzi https://github.com/keboola/platform-libraries/pull/135/files#r1268140531

Pointa je v tom, že ta classa má clientWrapper, kterej ale doteď nešel rozumně použít (obě použití by musely být oifováný) což bylo dost dementní, protože uvnitř této třídy šlo klienta získat buď $client nebo $this->clientWrapper, až na to si člověk musel pamatovat, že ten clientWrapper nesmí použít :)